Route/PathInfoBuilderSlashes.php

```diff
@@ -26,11 +26,12 @@ public function build(AdminInterface $admin, RouteCollection $collection)
     {
         $collection->add('list');
         $collection->add('create');
-        $collection->add('batch', null, array(), array('_method' => 'POST'));
+        $collection->add('batch', null, array(), array(), array(), '', array(), array('POST'));
```

Comment:
This looks a bit tricky: SonataAdminBundle requires [symfony/routing : ~2.3](https://github.com/sonata-project/SonataAdminBundle/blob/2.x/composer.json#L25) - the "_method" requirement was already deprecated in 2.2, but sonata admin does not seem to provide any alternative to set the route methods.

Implementing a BC check here would therefore not fix the deprecation for sonata admin 2.x - which means we might as well merge this into master?